src/python/gudhi/persistence_graphical_tools.py

```diff
@@ -72,11 +72,11 @@ def plot_persistence_barcode(
     """This function plots the persistence bar code from persistence values list
     , a np.array of shape (N x 2) (representing a diagram 
     in a single homology dimension), 
-    or from a :doc:`persistence file <fileformats>`.
+    or from a `persistence diagram <fileformats.html#persistence-diagram>`__ file.
```

Comment:
What does the double __ do? Do you need it because something strips one of them before is reaches the RST processor?

Reply:
Also, what is the advantage of this version compared to the one with `:doc:`, is it because it is shorter? Does it allow the `#label` thing while doc doesn't? Was the doc link broken? For uniformity? I don't care which one we use, just trying to understand the choices.

Reply:
Here, I wanted `persistence file` to link to the `Persistence diagram` section of  `fileformats` page.
If I say:
```
:doc:`persistence file <fileformats#persistence-diagram>`
or
:doc:fileformats#persistence-diagram
```
I have the following warning:
```
 WARNING: unknown document: fileformats#persistence-diagram
```

Reply:
Ok, good reason.